dev/com.ibm.websphere.appserver.features/test/src/com/ibm/ws/feature/utils/VersionlessFeatureDefinition.java

```diff
@@ -0,0 +1,141 @@
+package com.ibm.ws.feature.utils;
```

Comment:
You must add a copyright header to all source, doesn't matter if this is draft or not. Please update ASAP.

Reply:
copyright header added